src/util/deleteThought.ts

```diff
@@ -103,7 +102,7 @@ export const deleteThought = () => {
   }
 
   /** Sets the cursor or moves it back if it doesn't exist. */
-  const setCursorOrBack = (thoughtsRanked, { offset } = {}) => {
+  const setCursorOrBack = (thoughtsRanked: Array<Thought | Child | string> = [], { offset } = {offset: 0}) => {
```

Comment:
I think there are times when `offset` is undefined. If you set it to `0` it will change the behavior.